app/controllers/people_skills/filter_params.rb

```diff
@@ -40,7 +40,7 @@ def interest_of_row(row_id)
     end
 
     def skill_of_row(row_id)
-      skill_ids.nil? ? nil : skill_ids[row_id].to_i
+      skill_ids[row_id]&.to_i
```

Comment:
Hmmm I am not sure I understand the functionality 100% but are you sure this change is fine as it changes the behaviour a bit :thinking: